tests/internal/iframe-test.html

```diff
@@ -10,11 +10,11 @@
   <script src="test.js"></script>
 </head>
 <body onload="onload()">
-	<iframe src="iframe_sub.html" id="iframe-element" tabindex="0" class="container c2"></iframe>
-	<div tabindex="0" class="box b1" id="b1" style="left: 70px; top: -180px;"></div>
-	<div tabindex="0" class="box b1" id="b2" style="left: 180px; top: -120px;"></div>
+  <div tabindex="0" class="box b1" id="b1" style="left: 70px; top: 10px; margin-bottom: 20px;"></div>
+  <iframe src="iframe_sub.html" id="iframe-element" tabindex="0" class="container c2"></iframe>	
+	<div tabindex="0" class="box b1" id="b2" style="left: 120px; top: -150px;"></div>
 	<div tabindex="0" class="box b1" id="b3" style="left: 20px; top: -90px;"></div>
-	<div tabindex="0" class="box b1" id="b4" style="left: 190px; top: -50px;"></div>
+	<div tabindex="0" class="box b1" id="b4" style="left: 190px; top: -20px;"></div>
```

Comment:
Why such position change is needed? 
when it comes to fixing a bug, Modifying the TC should be avoided.

Reply:
Oh, I'll be careful about this next time. 
The reason for this is, I just simplified the test case and I thought the changing doesn't impact much about the test results.